packages/circuit-ui/styles/style-mixins.spec.tsx

```diff
@@ -359,6 +359,20 @@ describe('Style helpers', () => {
           "
       `);
     });
+
+    it('should match the snapshot with an inset outline', () => {
+      const { styles } = focusOutline('inset')({ theme: light });
+      expect(styles).toMatchInlineSnapshot(`
+        "
```

Comment:
Is the weird spacing in this template literal on purpose?

Reply:
Yes, it's because of the indentation in the source file. This template literal is generated by Jest.